### ISSUE TITLE
Changing fonts causes severe lag

### ISSUE THREAD

**Issue by DuckDuckWhale** [lapce/lapce#1032]
When typing the font name, the fonts change immediate after each letter after a very long lag time. I tried typing out the whole thing at once and it seems to be settled on a variable width font and doesn't actually reach the correct font even after waiting. Meanwhile the editor lags severely. Only after relaunching is the font correct.

Ubuntu 22.04, Nvidia, Wayland, a1148e4a1b1f04e50d29456e6b7383f218b73e6c

**Comment by DuckDuckWhale** [comment]
Looks like I filed a duplicate of #918.
